testimony/__init__.py

```diff
@@ -8,20 +8,20 @@
 
 from decimal import Decimal
 from testimony.constants import (
-    CLR_ERR, CLR_GOOD, CLR_RESOURCE, DOCSTRING_TAGS, PRINT_AUTO_TC,
+    CLR_ERR, CLR_GOOD, CLR_RESOURCE, DOC_TAGS, PRINT_AUTO_TC,
     PRINT_DOC_MISSING, PRINT_INVALID_DOC, PRINT_MANUAL_TC, PRINT_NO_DOC,
     PRINT_NO_MINIMUM_DOC, PRINT_NO_MINIMUM_DOC_TC,
     PRINT_PARSE_ERR, PRINT_TC_AFFECTED_BUGS, PRINT_TOTAL_TC, PRINT_REPORT,
-    SUMMARY_REPORT, VALIDATE_DOCSTRING_REPORT, BUGS_REPORT, MANUAL_REPORT,
+    SUMMARY_REPORT, VALIDATE_REPORT, BUGS_REPORT, MANUAL_REPORT,
     AUTO_REPORT)
 
 try:
     import termcolor
-    has_termcolor = True
-except ImportError, e:
-    has_termcolor = False
+    HAS_TERMCOLOR = True
+except ImportError, exception:
```

Comment:
Can you use the updated exception handling syntax?

    except ImportError as exception:

This is compatible with Python 2.6 and beyond. See section ["Compatibility Issues"](https://www.python.org/dev/peps/pep-3110/#compatibility-issues) in PEP 3110.